corecomponent/androidcomponent/src/main/res/navigation/navigation.xml

```diff
@@ -11,14 +11,23 @@
         <action
             android:id="@+id/action_session_to_session_detail"
             app:destination="@id/session_detail"
+            app:enterAnim="@anim/fade_in"
```

Comment:
Can we set default animation? 👀

Reply:
I tried use override default file like [this](https://github.com/android/architecture-components-samples/issues/537) but It not working...
Do you have any idea?

Reply:
Can you please add an issue that "defines the default animation"?

Reply:
I add an issue🙏🏻 #245